roles/matrix_mult/tasks/main.yml

```diff
@@ -0,0 +1,87 @@
+---
+- name: Ensure that NFD found nodes with GPU labels
+  # label list should be in sync with:
+  # https://github.com/NVIDIA/gpu-operator/blob/master/pkg/controller/clusterpolicy/state_manager.go#L26
+  shell:
+    (   oc get nodes -oname --ignore-not-found=false -l feature.node.kubernetes.io/pci-10de.present
+     || oc get nodes -oname --ignore-not-found=false -l feature.node.kubernetes.io/pci-0302_10de.present
+     || oc get nodes -oname --ignore-not-found=false -l feature.node.kubernetes.io/pci-0300_10de.present
+    ) | grep .
+
+- name: Get the list of nodes with GPUs
+  shell:
+    oc get nodes
+       -lnvidia.com/gpu.present=true
+       -o custom-columns=NAME:metadata.name
+       --no-headers
+    | cut -d. -f1
+  register: matrix_mult_gpu_nodes
+  failed_when: matrix_mult_gpu_nodes.stdout == ""
+
+- name: Create the entrypoint ConfigMap
+  command: oc apply -f "{{ matrix_mult_cm_entrypoint }}"
+
+- name: Create GPU matrix mult Pods
+  with_items: "{{ matrix_mult_gpu_nodes.stdout_lines }}"
+  shell: |
+    set -eo pipefail;
+    GPU_NODE_HOSTNAME=$(echo '{{ item }}' );
+    echo "Hostname: $GPU_NODE_HOSTNAME";
+    oc delete pod/matrix-mult-$GPU_NODE_HOSTNAME -n default --ignore-not-found=true
+    cat "{{ matrix_mult_pod }}" \
+      | sed "s|{{ '{{' }} gpu_node_hostname {{ '}}' }}|$GPU_NODE_HOSTNAME|" \
+      | sed "s|{{ '{{' }} matrix_mult_time {{ '}}' }}|{{ matrix_mult_time }}|" \
+      | oc apply -f-
+
+- name: "Let the GPU matrix mult Pods run {{ matrix_mult_time }}seconds"
+  command: "sleep {{ matrix_mult_time }}"
```

Comment:
if `matrix_mult_time` doesn't control the benchmark execution time, no need to sleep here